src/store/user/actions.js

```diff
@@ -0,0 +1,43 @@
+import { API_ENDPOINTS, STORAGE_KEYS, ENV } from '../../constants';
+import { LOGIN_USER, LOGOUT_USER } from './types';
+
+export const loginUserSuccess = (userData) => ({
+  type: LOGIN_USER,
+  payload: userData,
+});
+
+export const logoutUser = () => ({
+  type: LOGOUT_USER,
+});
+
+export const loginUser = (email, password) => async (dispatch) => {
+  try {
+    const response = await fetch(`${ENV}${API_ENDPOINTS.LOGIN}`, {
```

Comment:
I think the actual API call should lie in the `services.js` file, which is currently empty for you, and judging from this 1st criteria:
![Screenshot 2024-07-08 at 13 39 53](https://github.com/JoseCarlosBG/ReactTasks/assets/25910474/42055120-0d5c-4782-bbaa-0debe7475f31)